src/docker/Dockerfile

```diff
@@ -26,9 +26,6 @@ COPY python/requirements-app.txt ./
 RUN --mount=type=cache,target=/root/.cache  \
     { source activate rapids || echo ok ; } && pip install -r requirements-app.txt
 
-RUN --mount=type=cache,target=/root/.cache  \
-    { source activate rapids || echo ok ; } && pip install --no-deps streamlit pympler altair==4.2.2 watchdog
-
```

Comment:
- if removing watchdog, also remove from here: https://github.com/graphistry/graph-app-kit/blob/7353d0afb3870cf2eb0a4644953c88a24d4f7a89/src/python/util/log.py#L24
- removing pympler seems good
- I think we need altair for some demos, so that belongs in requirements-system.txt

Reply:
Those were all streamlit dependencies that I had to add manually when we switched to the `--no-deps` install option.

Reply:
so you can confirm altair is loading?

Reply:
```
$ docker exec -it docker_streamlit_1 pip list | grep altair
altair                        5.0.1
```